src/app/admin/authors/new/new.component.ts

```diff
@@ -29,7 +29,8 @@ export class NewAuthorComponent implements OnInit {
       return false;
     }
     return this.http.post(environment.api_endpoint + environment.single_author,
-      new HttpParams().set('name', this.currentAuthorName)).subscribe((res) =>
+      JSON.stringify({name: this.currentAuthorName}),
```

Comment:
Do you have to stringify it? [Angular htttp post examples](https://angular.io/guide/http#making-a-post-request)

Reply:
With the `Content-Type` setting, perhaps not; the `JSON.stringify` call was my first attempt to make sure that the data was being sent as JSON rather than form-encoded. And which overload of `http.post` is called determines how the arguments are interpreted.) But if it'll work without wrapping the object in this call, omitting it would indeed be better. If I can I'll test it this afternoon, and if you're right I'll make the change and update the PR.

Reply:
Okay, cool!

Reply:
As expected, it works just as well without the `JSON.stringify` call as with, so the PR now includes a commit removing the spurious calls.

Reply:
Excellent!